ssc/cmod_pvsamv1.cpp

```diff
@@ -1093,6 +1085,22 @@ void cm_pvsamv1::exec()
         if (!Simulation->annualSimulation)
             throw exec_error("pvsamv1", "The PV+Battery configuration requires a simulation period that is continuous over one or more years.");
 
+        int batt_forecast_choice = as_integer("batt_dispatch_wf_forecast_choice");
+        if (is_assigned("batt_pv_clipping_forecast")) {
+            p_pv_clipping_forecast = as_vector_ssc_number_t("batt_pv_clipping_forecast");
```

Comment:
I remember you asking about lifetime forecasts, but here`batt_pv_clipping_forecast` is 1 year only? Please specify in var table if its 1 yr forecast, if it isn't already